x-pack/plugin/autoscaling/src/main/java/org/elasticsearch/xpack/autoscaling/capacity/AutoscalingCalculateCapacityService.java

```diff
@@ -50,6 +50,12 @@ public AutoscalingCalculateCapacityService(Set<AutoscalingDeciderService> decide
 
     public void validate(AutoscalingPolicy policy) {
         policy.deciders().forEach((name, configuration) -> validate(name, configuration, policy.roles()));
+        SortedMap<String, Settings> deciders = addDefaultDeciders(policy);
+        if (deciders.isEmpty()) {
+            throw new IllegalArgumentException(
+                "no deciders enabled for policy [" + policy.name() + "] with roles [" + policy.roles() + "]"
```

Comment:
`enabled` -> `configured`?

Reply:
I went with `enabled` because `configured` would imply that the user did not configure it. But there are two contributions here: user configured and defaults. I can live with configured too though if you prefer it.

Reply:
I wonder if that's what we should say then: "no default nor user configured deciders for policy..."?

Reply:
Makes sense, fixed in 33e66221f954c706897287e419e889e00b68601d

Reply:
LGTM. Thanks.